docs/development/compiling/prgenv.md

```diff
@@ -119,8 +119,7 @@ also be included. See [here][2.2] for more information.
     systems like the `mpicc`, `mpic++` and `mpif90` wrappers. You don't need to
     use these wrappers to compile an MPI code on LUMI. See [here][3].
 
-Below are examples on how to use the wrappers for the different programming
-languages.
+Below are examples how to use the wrappers for the different programming languages.
```

Comment:
For me, the old wording was correct

```suggestion
Below are examples on how to use the wrappers for the different programming languages.
```

Reply:
The `on` is superfluous, as far as I understand.